client/src/__tests__/PrescriptionCreation/index.spec.tsx

```diff
@@ -0,0 +1,222 @@
+import React from 'react';
+import {
+  render, screen, waitFor,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+import { setupServer } from 'msw/node';
+import { rest } from 'msw';
+import AppTest from '../../AppTest';
+import PatientSubmission from '../../components/screens/PatientSubmission';
+import { ResponseBuilder } from '../utils/Utils';
+
+describe('PrescriptionCreation', () => {
+  const server = setupServer();
+
+  beforeAll(() => {
+    server.listen({ onUnhandledRequest: 'error' });
+  });
+
+  beforeEach(() => {
+    server.resetHandlers();
+  });
+
+  afterAll(() => {
+    server.close();
+  });
+
+  test('Enable/Disable Save Prescription Button', async () => {
+    render(
+      <AppTest>
+        <PatientSubmission />
+      </AppTest>,
+    );
+
+    expect(screen.getByText(/compléter plus tard/i).closest('button')).toBeDisabled();
+
+    const mrnOptions = screen.getByPlaceholderText(/Sélectionner un dossier/i);
```

Comment:
Il faut faire attention quand on crée des variables comme ça. Si on les réutilises plus tard dans le test, ça ne sera plus "in sync"